tests/tasks/setup-802_1x_server.yml

```
@@ -0,0 +1,79 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Install EPEL on enterprise Linux for hostapd
+  command: yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm
+  args:
+    warn: false
+    creates: /etc/yum.repos.d/epel.repo
+  when:
+    - ansible_distribution in ['RedHat', 'CentOS']
+
+- name: Install hostapd
+  package:
+    name: hostapd
+    state: present
+
+- name: Copy server certificates
+  copy:
+    src: "{{ item }}"
+    dest: "/etc/pki/tls/{{ item }}"
+  with_items:
+    - server.key
+    - dh.pem
+    - server.pem
+    - cacert.pem
+
+- name: Create test interfaces
+  shell: |
+    ip link add veth1 type veth peer name veth1-br
+    ip link add veth2 type veth peer name veth2-br
+
+    ip link add br1 type bridge
+    ip link set br1 up
+
+    ip netns add ns1
+
+    ip link set veth1 netns ns1
+
+    ip netns exec ns1 ip addr add 10.1.1.11/24 dev veth1
```

Please use IP addresses from https://tools.ietf.org/html/rfc5737 to avoid conflicts with local networks.

```suggestion
    ip netns exec ns1 ip addr add 203.0.113.0/24 dev veth1
```

---

Haven't heard of this RFC before! fixed now so all ips in the 802.1x tests use the 203.0.113.2/24 subnet